### PR TITLE
Initial support for themes.

### PR DESCRIPTION
**Pull request by TimBarham**
A themes can be specified via the --theme commandline switch or a theme property on the options API property. It can be a file name pointing to a JSON file, or a JavaScript object or JSON string. The format is as specified [here](https://github.com/Microsoft/cordova-simulate/wiki/Support-for-Themes).

### CODE REVIEW

**Comment by MSLaguana** [comment]
Looks good to me.

File: src/server/theme.js [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+    // If selector is an array of selectors, we will output them as separate rules. This is important when selectors
+    // contain browser specific bits that would break the overall selector in other browsers.
+    if (!Array.isArray(selector)) {
+        selector = [selector];
+    }
+
+    // Construct the property bundle
+    var propertyBundle = [];
+    Object.keys(properties).forEach(function (propertyName) {
+        propertyBundle.push('  ' + propertyName + ': ' + properties[propertyName] + ';')
```
Comment (TimBarham) [inline src/server/theme.js]: Because that doesn't give me a semicolon on the last one 😄

**Comment by albertinad** [comment]
Looks good to me!

**Comment by TimBarham** [comment]
Thanks guys!

**Pull request merged**